cmd/spruce/main_test.go

```diff
@@ -162,7 +162,8 @@ func TestMain(t *testing.T) {
 			stdout = fmt.Sprintf(format, args...)
 		}
 		var stderr string
-		printfStdErr = func(format string, args ...interface{}) {
+		//Edit log stderr function
+		PrintfStdErr = func(format string, args ...interface{}) {
```

Comment:
I'm confused why you did this instead of editing the existing printfStdErr function, which was put in place to do tests just like this